### ISSUE TITLE
Add endpoint example accessing data

### ISSUE THREAD

**Issue by wellingguzman** [directus/docs#84]
Add an example of a custom endpoint accessing data from collections

**Comment by Pujan789556** [comment]
I have tried to create a custom endpoint following [https://github.com/directus/docs/blob/master/api/data.md](url)

But got the error: **{"error":{"code":300,"message":"Reading items from \"COLLECTION_NAME\" collection was denied"}}**

I had set permission to read collection data for **public** user role.

**Comment by denveloperj** [comment]
I'm having the same issue at the moment. 

Trying to find an email address in my custom `members` table but all that it returns is `Reading items from "COLLECTION_NAME" collection was denied`

**Comment by benhaynes** [comment]
Thoughts @bjgajjar?

**Comment by andreasanta** [comment]
I have the same issue, example below, very simple custom endpoint. The read permission for the collection is set to public.

```
use Directus\Application\Http\Request;
use Directus\Application\Http\Response;
use Directus\Services\ItemsService;

return [
    '' => [
        'method' => 'GET',
        'handler' => function (Request $request, Response $response) {
            
            // Get all answers from DB
            $itemsService = new ItemsService($this);

            $params = $request->getQueryParams();
            $programs = $itemsService->findAll('programs', $params);


            return $response->withJson([
                'data' => [
                    $programs
                ]
            ]);
        }
    ],
];

```

**Comment by benhaynes** [comment]
🔔  @hemratna @theharshin @bjgajjar

**Comment by bjgajjar** [comment]
@benhaynes 

I'll check this and update you ASAP!

**Comment by bjgajjar** [comment]
@andreasanta Thanks for the help!  Fixes in #910

**Comment by rijkvanzanten** [comment]
I'm reopening this as it was never added to the docs.. (or I can't find it)

@bjgajjar
